scripts/operations_tests/operations_tests.py

```diff
@@ -197,6 +212,8 @@ def run_test(test_case):
             print(".", end="")
         elif test_case.status == "fail":
             print("F", end="")
+            if hasattr(test_case, 'known_failure') and test_case.known_failure:
+                print(f"\n\"known_failure\": \"{test_case.known_failure}\"")
```

Comment:
I don't think the second part of the if is needed, and the print could be simpler:
```python
if hasattr(test_case, 'known_failure'):
                print(f'\nKnown failure: "{test_case.known_failure}"')
```